fingerpaintview/src/main/java/tech/picnic/fingerpaintview/FingerPaintImageView.kt

```diff
@@ -162,7 +162,7 @@ class FingerPaintImageView @JvmOverloads constructor(context: Context,
 
         // make sure drawings are kept within the image bounds
         if (imageBounds.contains(event.x, event.y)) {
-            paths.add(Path().also { it.moveTo(event.x, event.y) } to Paint(pathPaint))
+            paths.add(Path().also { it.moveTo(event.x+1, event.y+1) } to Paint(pathPaint))
```

Comment:
There should be whitespaces around each `+` (default Android Studio formatting should do the trick)